Client/Frontend/Fakespot/FakespotViewController.swift

```diff
@@ -131,6 +133,30 @@ class FakespotViewController:
         setupView()
         listenForThemeChange(view)
         viewModel.fetchProductIfOptedIn()
+
+        subscribeToRedux()
+    }
+
+    // MARK: - Redux
+
+    func subscribeToRedux() {
```

Comment:
We should add a typealias for the SubscriberStateType like in BrowserViewController:

```typealias SubscriberStateType = BrowserViewControllerState```